### PR TITLE
[bundle] Remove sanitize-html/postcss/source-map from client bundles.

### PR DESCRIPTION
**Pull request by alloy**
Replaces [a previously added dependency](https://github.com/artsy/force/pull/4254) and its transitive dependencies with [a leaner one](https://github.com/bevacqua/insane).

### Before

Screenshot 2019-10-10 at 16 37 08

### After

Screenshot 2019-10-10 at 16 34 08

### CODE REVIEW

File: src/desktop/models/page.coffee [hunk 1]
```diff
@@ -11,7 +11,5 @@ module.exports = class Page extends Backbone.Model
   urlRoot: "#{sd.API_URL}/api/v1/page"
 
   sanitizedHtml: (attr) ->
-    clean = sanitizeHtml(@get(attr), {
-      allowedTags: sanitizeHtml.defaults.allowedTags.concat([ 'h1', 'h2', 'summary', 'details' ])
-    })
+    clean = insane(@get(attr))
```
Comment (sweir27) [inline src/desktop/models/page.coffee]: @alloy can you verify (with screenshot, etc.) that we're still properly rendering pages that show markdown (like https://staging.artsy.net/conditions-of-sale)?

Also, does `insane` have its own definition of what tags are allowed? Does it include those above?

Comment (alloy) [reply src/desktop/models/page.coffee]: Good call, I was relying on the tests to convey that this was still the case.

Screenshot 2019-10-10 at 19 11 19

Comment (alloy) [reply src/desktop/models/page.coffee]: As for the allowed tags, it has the same as sanitized-html _plus_ the ones you previously added manually https://github.com/bevacqua/insane/blob/v2.6.1/defaults.js#L12-L15

Comment (sweir27) [reply src/desktop/models/page.coffee]: Seems correct!

I remember looking at `insane` for this but I can't remember why I decided to go with `sanitize-html`. Should have done a better job documenting the alternatives.

**Comment by sweir27** [comment]
Looks good! Not sure if you care about that codecov warning, otherwise feel free to merge.

**Comment by alloy** [comment]
Nope, I didn’t do anything to reduce coverage 🤷‍♂

File: src/desktop/models/page.coffee (hunk 1 above)
Comment (zephraph) [reply src/desktop/models/page.coffee]: The dependency RFC process helps with this!

**Comment by eessex** [comment]
The codecov check is on the coverage of the diff, which with the existing tests is 50%. I’d take this as a suggestion that this function could use a test, but not blocking.

**Comment by alloy** [comment]
@eessex Gotcha, makes sense, thanks!

**Pull request merged**